addons/rose/addon/components/rose/form/radio/radio/index.hbs

```diff
@@ -12,7 +12,7 @@
     @disabled={{@disabled}}
     @changed={{@changed}} />
 
-  <label class="rose-form-radio-label" for={{this.id}}>
+  <label class="rose-form-radio-label{{if @helperText " helper-text"}}" for={{this.id}}>
```

Comment:
Is there a way to avoid adding another class here?

Reply:
yes! works w/o class.